Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM ubuntu:20.04
-
+ARG debug
```

Comment:
I'm concerned about adding a `debug` argument to the primary Dockerfile for this project without tying it to the deeper strategy for debugging. Is this an indication that you should be using the [dev Dockerfile](https://github.com/cyberark/conjur/blob/master/dev/Dockerfile.dev) instead? Do we expect end-users to leverage this `debug` flag? In my opinion, the main Dockerfile should be clean and representative of the primary final image we build for end users that gets published to Dockerhub, etc.